cli/boards.go

```diff
@@ -374,9 +318,12 @@ func downloadFile(ctx context.Context, filepath, url string) error {
 	if err != nil {
 		return errors.Wrap(err, "error downloading the requested package")
 	}
-
 	defer utils.UncheckedErrorFunc(resp.Body.Close)
 
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("invalid status code %q for url %q", resp.Status, url)
+	}
+
```

Comment:
Tidiness. This came up for another app ticket. Without this check, we attempt to untar the http error message and fail.